docs/src/components/ReanimatedAnimations/ReanimatedAnimationsSection/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import CodeBlock from '@theme/CodeBlock';
+import styles from './styles.module.css';
+import InteractiveExampleComponent from '../../InteractiveExample/InteractiveExampleComponent';
+import ArrowRight from '@site/static/img/examples/arrow-right-light.svg';
+
+interface Props {
+  title: string;
+  body: string;
+  docsLink: string;
+  component: React.ReactNode;
+  mobileComponent: React.ReactNode;
+  tabletComponent: React.ReactNode;
+  code: string;
+  idx: number;
+}
+
+export default function ReanimatedAnimationsSection({
+  title,
+  body,
+  docsLink,
+  component,
+  mobileComponent,
+  tabletComponent,
+  code,
+  idx,
+}: Props) {
+  const [isMobileView, setIsMobileView] = useState(window.innerWidth <= 420);
+  const [isTabletView, setIsTabletView] = useState(window.innerWidth <= 860);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobileView(window.innerWidth <= 420);
+      setIsTabletView(window.innerWidth <= 860 && window.innerWidth > 420);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  return (
+    <div className={styles.animationsSectionWrapper}>
+      {mobileComponent !== null && !!isMobileView && !isTabletView && (
```

Comment:
```suggestion
      {mobileComponent !== null && isMobileView && !isTabletView && (
```
it seems like there's no need to convert boolean to boolean with `!!`